{"name": "test_update_user", "status": "failed", "statusDetails": {"message": "AssertionError: 更新失败: 用户不存在\nassert -1 == 0", "trace": "get_base_url = 'http://localhost:8080', get_token = ''\n\n    def test_update_user(get_base_url, get_token):\n        \"\"\"更新用户信息\"\"\"\n        user_id = read_yaml(\"user_id\")\n        url = f\"{get_base_url}/user/updateUser\"\n        headers = {\"Authorization\": f\"Bearer {get_token}\"}\n        data = {\n            \"id\": user_id,\n            \"name\": \"test001_new\",\n            \"password\": \"654321\",\n            \"phone\": \"13800000000\",\n            \"email\": \"test001@abc.com\"\n        }\n    \n        resp = requests.put(url, headers=headers, data=data)\n        print(resp.text)\n        j = resp.json()\n        assert resp.status_code == 200\n>       assert j[\"code\"] == 0, f\"更新失败: {j['message']}\"\nE       AssertionError: 更新失败: 用户不存在\nE       assert -1 == 0\n\ntestcases\\test_user_update.py:22: AssertionError"}, "description": "更新用户信息", "start": 1761030489542, "stop": 1761030489593, "uuid": "7292977c-f4a0-411a-b716-7f07b26ef833", "historyId": "f73ab9f1c16a49f75d94bf0fd5db6533", "testCaseId": "f73ab9f1c16a49f75d94bf0fd5db6533", "fullName": "testcases.test_user_update#test_update_user", "labels": [{"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "test_user_update"}, {"name": "host", "value": "DESKTOP-UGQEI6T"}, {"name": "thread", "value": "26396-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.test_user_update"}], "titlePath": ["testcases", "test_user_update.py"]}